src/w3c/defaults.js

```diff
@@ -62,6 +64,7 @@ const w3cDefaults = {
     "no-http-props": true,
     "check-punctuation": false,
     "local-refs-exist": true,
+    "internal-slot-warn": true,
```

Comment:
We probably only want to enable this by default once we beta test it in a few specs.